MemeMe/MemeDetailViewController.swift

```diff
@@ -1,47 +1,43 @@
 //
-//  ViewController.swift
+//  MemeDetailViewController.swift
 //  MemeMe1.0
 //
 //  Created by Waylon Kumpe on 7/11/22.
 //
 
 import UIKit
 
-class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate, UITextFieldDelegate {
-// MARK: - ImageView.
-    @IBOutlet weak var imageView: UIImageView!
-    var memedImage: UIImage!
+class MemeDetailViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate, UITextFieldDelegate {
 
-    // MARK: TextFields.
+    // MARK: Properties
+    var memedImage: UIImage!
+    @IBOutlet weak var imageView: UIImageView!
     @IBOutlet weak var topTextField: UITextField!
     @IBOutlet weak var bottomTextField: UITextField!
-
-    // MARK: ToolBars.
     @IBOutlet weak var bottomToolBar: UIToolbar!
     @IBOutlet weak var topToolBar: UIToolbar!
-
-    // MARK: Buttons.
     @IBOutlet weak var cameraButton: UIBarButtonItem!
     @IBOutlet weak var albumButton: UIBarButtonItem!
     @IBOutlet weak var shareButton: UIBarButtonItem!
 
-    // MARK: - Take a new photo with the camera.
+    // MARK: Take or choose a photo
+    // Take a new photo with the camera.
```

Comment:
2 line comments should use /**/ instead of //.
Example:
/* MARK: Take or choose a photo
Take a new photo with the camera. */